docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,89 @@
+Manual Testing
+==============
+
+This document describes a high-level script of manual tests to check for. We
+should aim to replace items on this list with automated Spectron test cases.
```

Comment:
...and presumably these tests should also be run on all OS and Arch combinations, i.e. Windows 32-bit, Windows 64-bit, Linux 32-bit, Linux 64-bit, macOS 64-bit? Which probably means using virtual machines to properly check that the 32-bit builds don't accidentally contain any 64-bit code. (The VM images from http://modern.ie are very useful for this, which will also allow us to test Windows 7, Windows 8 and Windows 10. And I guess testing in a VM also allows us to test that Etcher runs on computers that don't already have NodeJS installed :wink: )

Reply:
Yep, the CI services builds should take care of this pretty well.